bower.json

```diff
@@ -20,6 +20,7 @@
     "moment-timezone": "0.5.3",
     "patternfly": "3.43.0",
     "jquery": "3.2.1",
+    "jquery-qrcode": "https://github.com/david-martin/jquery-qrcode.git#v1.0.0",
```

Comment:
Did you test whether this library works from the upstream repo? I can't remember why we use Daves fork for this.

Reply:
@sedroche changed to '*' as 1.0.0 could not be find